cmd/kyma/alpha/deploy/opts.go

```diff
@@ -70,3 +76,11 @@ func (o *Options) validateFilters() error {
 	o.Filters = filters
 	return nil
 }
+
+func (o *Options) validateTarget() error {
+	if o.Target == "control-plane" || o.Target == "remote" {
```

Comment:
I suggest we define these two values as const, so that the following returned errors can also reuse, keeping the value consistent.